mathics/builtin/box/inout.py

```diff
@@ -63,6 +90,7 @@ class StyleBox(Builtin):
     </dl>
     """
 
+    summary_text = "associates boxes with styles"
```

Comment:
I know it's a kind of trivial thing. but could we put this after "attributes" and have the class names in sorted order. That's the form that all of the others were in. 

Thanks in advance.

Reply:
I did this in this file. In general, I put the summary_text just after the docstring for convenience during the edition. Once everything has the attributes, I can think in implement something like "black" that puts all the attributes in order and performs some automatic formatting in the modules.